projects/plugins/jetpack/readme.txt

```diff
@@ -242,39 +242,22 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 11.0-beta - 2022-05-31
+### 11.1-a.1 - 2022-06-08
+#### Major Enhancements
+- Blocks: introduce a new Recipe block to display an easy to read recipe anywhere on your site.
```

Comment:
I wonder if we should leave this out of the "major" section for now, since the block is still in Beta?

Reply:
Ah wasn't aware, just drop it out for now?

Reply:
How does https://github.com/Automattic/jetpack/pull/24674/commits/68d2f4d59e2a9bb8b200661a390876d81dbcfc6b look?

Reply:
That works for me. 👍